packages/imtr/src/util.ts

```diff
@@ -1,5 +1,17 @@
+import { createHash } from "./deps.ts";
+
 export const strFmt = (str: string) => str.trim();
-export const format = (el: string | undefined) => typeof el === 'string' ? strFmt(el) : el;
+export const format = (el: string | undefined) =>
+  typeof el === "string" ? strFmt(el) : el;
+
+/**
+ * Consistent hashing for id's
+ */
+export const getId = (input: any): string => {
+  const hash = createHash("md5");
+  hash.update(JSON.stringify(input));
+  return hash.toString();
+};
```

Comment:
Basic hash generation used by postprocessor in transform step.

Reply:
Very simple 💪🏽